scripts/services/hero2/hero2-rgbd-shm-server.service

```diff
@@ -6,7 +6,7 @@ WantedBy=multi-user.target
 
 [Service]
 User=amigo
-Restart=on-failure
+Restart=always
```

Comment:
@MatthijsBurgh @ar13pit discuss this change

Reply:
We should keep this. There should not be a reason for the shm server to exit cleanly as you want it to run continuously

Reply:
Yes, we should keep this. Especially with the pending RGBD changes, this is required.